Producer/python/utils/makeFatJets_cff.py

```diff
@@ -88,13 +89,8 @@ def initFatJets(process, isData, labels):
                 )
             )
 
-    # Charged hadron subtraction
-    addattr('pfCHS',
-        cms.EDFilter("CandPtrSelector",
-            src = cms.InputTag(pfSource),
-            cut = cms.string("fromPV")
-        )
-    )
+    # Add PF CHS
+    sequence += chsSequence(process, pfSource, skipIfExists = True)
```

Comment:
Won't this not add pfCHS to the sequence since skipIfExists = True and the other bit is in prod.py?

Reply:
You are right. Updating the PR now